src/sentry/static/sentry/app/components/splitDiff.tsx

```diff
@@ -0,0 +1,121 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+import {diffChars, diffWords, diffLines, Change} from 'diff';
+
+const diffFnMap = {
+  chars: diffChars,
+  words: diffWords,
+  lines: diffLines,
+};
+
+type Props = {
+  base: string;
+  target: string;
+  type: keyof typeof diffFnMap;
+  className: string;
+};
+
+export function SplitDiff({className, type, base, target}: Props) {
```

Comment:
I believe you can remove this because you already provide a default export

Reply:
woops